website/docs/reference/dbt-jinja-functions.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "dbt Jinja Functions"
+---
+
+In addition to the standard Jinja library ([docs](https://jinja.palletsprojects.com/en/2.11.x/templates/)), we've added additional functions and variables to the Jinja context that are useful when working with a dbt project:
+* [adapter](adapter)
```

Comment:
this is great! I bet we can run a for-loop over this list of jinja functions to generate this list in the future.

For the moment though, we should specify a more complete path for each of these entries. This is honestly a bug with our `<Link>` component: a link like `[adapter](adapter)` unfortunately links to `/building-a-new-adapter/`. We can and should fix that bug separately from getting this merged.

For reference: the problem is that we do a suffix match on the path. We happen to find `building-a-new-adapter` before we find `adapter` when iterating over the documents in this repo, and so that's the link that gets rendered.

Reply:
Oooooh that's wild! I assumed it searched by "parts" of the path, not suffixes. Ok that explains some of the behavior I've seen in other places!

Reply:
I'll work around it for now, but it would be good to do this in the future

Reply:
And separately, it would definitely be cool to iterate this list automatically!